tests/unit/modules/test_virt.py

```diff
@@ -1358,14 +1358,15 @@ def test_update(self):
         }
 
         # Update with boot parameter case
-        self.assertEqual(
-            {
-                "definition": True,
-                "disk": {"attached": [], "detached": []},
-                "interface": {"attached": [], "detached": []},
-            },
-            virt.update("my vm", boot=boot),
-        )
+        self.assertEqual({
+                'definition': True,
```

Comment:
black should replace the `'` by `"` here.

Reply:
☝️

Reply:
Thanks waynew.  I just ran pre-commit and did a push.  :)